compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/QueryDesugar.java

```diff
@@ -1349,6 +1378,11 @@ public void visit(BLangFieldBasedAccess fieldAccessExpr) {
         fieldAccessExpr.expr.accept(this);
     }
 
+    @Override
+    public void visit(BLangFieldBasedAccess.BLangStructFunctionVarRef structFunctionVarRef) {
+        structFunctionVarRef.expr.accept(this);
+    }
```

Comment:
Uncovered visits within QueryDesugar mean we don't have tests with such statements being used inside a query action/expression. Better to create an issue to track this adding tests

Reply:
Created https://github.com/ballerina-platform/ballerina-lang/issues/31658